src/components/manage/Delete/Delete.jsx

```diff
@@ -32,172 +27,110 @@ const messages = defineMessages({
   },
 });
 
-/**
- * Delete container class.
- * @class Delete
- * @extends Component
- */
-class Delete extends Component {
-  /**
-   * Property types.
-   * @property {Object} propTypes Property types.
-   * @static
-   */
-  static propTypes = {
-    deleteContent: PropTypes.func.isRequired,
-    getContent: PropTypes.func.isRequired,
-    deleteRequest: PropTypes.shape({
-      loading: PropTypes.bool,
-      loaded: PropTypes.bool,
-    }).isRequired,
-    pathname: PropTypes.string.isRequired,
-    content: PropTypes.shape({
-      title: PropTypes.string,
-    }),
-    returnUrl: PropTypes.string,
-  };
+const Delete = ({ location, history }) => {
+  const dispatch = useDispatch();
+  const intl = useIntl();
+  const [isClient, setisClient] = useState(false);
```

Comment:
You can move isClient logic to a new hook as suggested by @JeffersonBledsoe

Reply:
I was thinking of creating a separate PR , if everything is OK then it can be merged and I can pull the change in other working branches